### PR TITLE
1835 update City description

### PR DESCRIPTION
**Pull request by volker18xxdev**
Update city description, capitalisation to incremental, delete buying tunnel/bridges

### CODE REVIEW

**Comment by jenf** [comment]
please ensure rubocop passes before creating PRs (see wiki for devs on how to autofix most of them)

**Comment by volker18xxdev** [comment]
rubocop checks are now ok.

**Comment by tobymao** [comment]
failing

File: lib/engine/game/g_1835/meta.rb [hunk 1]
```diff
@@ -16,6 +16,13 @@ module Meta
         GAME_RULES_URL = 'http://google.com'
 
         PLAYER_RANGE = [3, 7].freeze
+        OPTIONAL_RULES = [
+          {
+            sym: :Clemens,
+            short_name: 'Clemens',
+            desc: 'all Privates and minors are available, 4-3-2-1-1-2-3-4, Minors start when Bay 50%',
```
Comment (benjaminxscott) [inline lib/engine/game/g_1835/meta.rb:23]: presumably "Buy" instead of "Bay" ?

Comment (volker18xxdev) [reply lib/engine/game/g_1835/meta.rb]: no, minors will only start, if the Bayern was floated.

Comment (benjaminxscott) [reply lib/engine/game/g_1835/meta.rb]: Ah is that the Bayrische Eisenbahn?

Comment (volker18xxdev) [reply lib/engine/game/g_1835/meta.rb]: yes.

Comment (benjaminxscott) [reply lib/engine/game/g_1835/meta.rb]: May want to use the full name, for clarity

File: lib/engine/game/g_1835/meta.rb [hunk 2]
```diff
@@ -16,6 +16,13 @@ module Meta
         GAME_RULES_URL = 'http://google.com'
 
         PLAYER_RANGE = [3, 7].freeze
+        OPTIONAL_RULES = [
+          {
+            sym: :Clemens,
```
Comment (jenf) [inline lib/engine/game/g_1835/meta.rb:21]: style - we tend to keep symbols snake case

Comment (volker18xxdev) [reply lib/engine/game/g_1835/meta.rb:21]: ok, i'll correct it tomorrow.

**Comment by volker18xxdev** [comment]
i'll update the snake case and description in meta. Update the company share definition.

File: lib/engine/game/g_1835/game.rb [hunk 3]
```diff
@@ -483,6 +585,16 @@ class Game < Game::Base
             logo: '1835/PR',
             simple_logo: '1835/PR.alt',
             tokens: [0],
+            abilities: [
+              {
+                type: 'exchange',
+                corporations: %w[PR_12],
+                owner_type: 'player',
+                when: ['Phase 2.3', 'Phase 2.4', 'Phase 3.1'],
```
Comment (jenf) [inline lib/engine/game/g_1835/game.rb:593]: I'm not entirely sure this when is right, but can be fixed later.

File: lib/engine/game/g_1835/game.rb [hunk 4]
```diff
@@ -25,14 +25,18 @@ class Game < Game::Base
                         brown: '#7b352a')
 
         CURRENCY_FORMAT_STR = '%dM'
+        # game end current or, when the bank is empty
+        GAME_END_CHECK = { bank: :current_or }.freeze
```
Comment (jenf) [inline lib/engine/game/g_1835/game.rb:29]: There's no bankrupcy check here, so the game will continue with zero players left? (See 1846 on how to handle multiple bankrupcies)

Can be fixed later.

**Review by jenf — APPROVED** [review]

**Pull request merged**